app/controllers/users_controller.rb

```diff
@@ -6,12 +6,12 @@ class UsersController < ApplicationController
   def show
     @can_edit = can_edit?
     if current_user.id == @user.id
-      @my_datasets = Dataset.my_datasets(current_user)
-      @awaiting_datasets = Dataset.admin_awaiting_datasets(current_user)
-      @withdrawn_datasets = Dataset.admin_withdrawn_datasets(current_user)
+      @my_works = Work.my_works(current_user)
```

Comment:
👍 polymorphism may also be used here should it ever be determined that some object less generic than `Work` need be used.